pom.xml

```diff
@@ -544,16 +544,23 @@
                 <version>${maven-pmd.version}</version>
                 <executions>
                     <execution>
-                        <phase>test</phase>
+                        <phase>compile</phase>
+<!--                        <goals>-->
+<!--                            <goal>check</goal>-->
+<!--                            <goal>cpd-check</goal>-->
+<!--                        </goals>-->
                     </execution>
                 </executions>
                 <configuration>
+                    <linkXRef>false</linkXRef>
```

Comment:
what's this one for?

Reply:
it can be used to add the errors in the reports. take a look at http://maven.apache.org/jxr/maven-jxr-plugin/